### ISSUE TITLE
Resize: Opposite of Smart

### ISSUE THREAD

**Issue by wesbos** [nikitabobko/AeroSpace#741]
Adding resizing shortcuts, I really like the smart resize option that will pick width or height depending on layout:

Finding myself with 4 commands for small/large smart resizes.

```
up = 'resize smart +70'
shift-up = 'resize smart +20'
down = 'resize smart -70'
shift-down = 'resize smart -20'
```

Now I want to add the same commands for the opposite of whatever smart is.

I could explicitly add 8 more keyboard commands for width/height, but what I really want is something that would just smart resize the opposite axis like this:

```
cmd-down = 'resize !smart -20'
```

This might be a dumb request so feel free to close, but I'm finding myself making (and remembering) a million shortcuts

**Comment by nikitabobko** [comment]
I don't mind adding `smart-opposite` if anyone wants to bring a PR, should be easy to implement

**Comment by nikitabobko** [comment]
Released in 0.16.0-Beta
